src/components/Visualizations/Table/KDA.jsx

```diff
@@ -5,16 +5,16 @@ import palette from 'components/palette.css';
 import strings from 'lang';
 import styles from './KDA.css';
 
-const KDA = ({ kills, deaths, assists, matchId = uuid.v4() }) => {
+const KDA = ({ kills, deaths, assists, matchId = uuid.v4(), together }) => {
   const kdaSum = kills + deaths + assists;
 
   return (
     <div className={styles.container}>
       <div>
-        {kills}
+        {together ? `${kills}/${deaths}/${assists}` : kills}
       </div>
       <div>
-        <div className={styles.percent} data-tip data-for={`kda-${matchId}`}>
+        <div className={styles.percent} data-tip data-for={`kda-${matchId}`} style={{ width: together && '100%' }}>
```

Comment:
Could you do a ternary on className instead of using in line styles? So compose the existing class in the css to a class called percentTogether which overrides width to 100%,  then do a ternary on the classname